docs/flinkDev/building.md

```diff
@@ -97,35 +97,49 @@ mvn clean install
 
 ## Hadoop Versions
 
-{% info %} Most users do not need to do this manually. The [download page]({{ site.download_url }}) contains binary packages for common Hadoop versions.
+Flink has optional dependencies to HDFS and YARN which are both dependencies from [Apache Hadoop](http://hadoop.apache.org). There exist many different versions of Hadoop (from both the upstream project and the different Hadoop distributions). If you are using a wrong combination of versions, exceptions can occur.
 
-Flink has dependencies to HDFS and YARN which are both dependencies from [Apache Hadoop](http://hadoop.apache.org). There exist many different versions of Hadoop (from both the upstream project and the different Hadoop distributions). If you are using a wrong combination of versions, exceptions can occur.
+Flink can be built against any Hadoop version >= 2.4.0, but depending on the version it may be a 1 or 2 step process.
 
-Hadoop is only supported from version 2.4.0 upwards.
-You can also specify a specific Hadoop version to build against:
+### Pre-bundled versions
+
+If you want to build against Hadoop 2.4.1, 2.6.5, 2.7.5 or 2.8.3, then it is sufficient to run:
+
+{% highlight bash %}
+mvn clean install -DskipTests -Dhadoop.version=2.6.5
+{% endhighlight %}
+
+To further package a shaded pre-packaged hadoop jar into the distribution, additionally activate the `include-hadoop` profile`:
```

Comment:
does it not have the `lib` details on purpose now? the old version seems to be more clear:
`If you want to build a Flink distribution that has a shaded Hadoop jars, pre-packaged in the lib folder, additionally activate the include-hadoop profile`:`